node-operators/validators/onboarding/account-setup.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Set Up an Operator Account
+description: Learn how to map your node's account with your stash account, making your node eligible to secure Tanssi and the Tanssi-powered networks and receive rewards.
+icon: octicons-tools-24
+---
+
+# Set Up Your Account to Secure the Tanssi Ecosystem
+
+## Introduction {: #introduction }
+
+Before you can start participating in the Tanssi protocol, validating blocks, and securing the Tanssi ecosystem, you'll need to set up your account and establish your node's eligibility.
+
+This step maps your Tanssi stash account (the one keeping track of your rewards) to your node's session keys (the ones used for communication and consensus), allowing the protocol to include your node in the active set. 
+
+By following the steps outlined in this guide, you'll learn everything you need to know to validate within the Tanssi ecosystem.
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+Before opting into a Tanssi-enabled vault and the Tanssi network, make sure that:
+
+- You have a node correctly [set up and running](/node-operators/validators/onboarding/run-a-validator/){target=\_blank}
+- You [have registered as an operator](/node-operators/validators/onboarding/register-in-symbiotic/){target=\_blank} in the Symbiotic registry 
+- You [have opted in to Tanssi Network and a Tanssi-enabled vault](/node-operators/validators/onboarding/opt-in-to-tanssi/){target=\_blank}
+
+## Map an Account to Your Node {: #map-account }
+
+The first step is a two-step process that generates and maps the session keys to your account. Session keys can be compared to the node's ID, and are used to perform network operations, such as signing validity proofs, whereas your account keeps track of your work and related rewards, and could have an on-chain identity. By mapping the session key to your account, you create an association between your account and your node.
+
+You will need to create session keys for your primary and backup servers. Each server should have its own 
+unique keys. Since the keys never leave your servers, you can consider them a unique ID for that server.
```

Comment:
```suggestion
You will need to create session keys for your primary and backup servers. Each server should have its own unique keys. Since the keys never leave your servers, you can consider them a unique ID for that server.
```